docs/actions/user.update_self.md

```diff
@@ -14,5 +16,9 @@ Updates the request user. Removes starting and trailing spaces from `username`.
 
 The given `gender` must be present in `organization/genders`.
 
+`meeting_id` is only for editing meeting-internal data, and the value will be thrown away afterwards.
```

Comment:
How about:
`meeting_id` is only used for processing meeting-internal data and will be omitted otherwise.

Reply:
The purpose of my description was to point out that `meeting_id` won't be written into the data anywhere and is just additional information given to aid action-internal calculations.
Your proposition doesn't do that, so I won't change it-